package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "auditory-haptic-graphics-browser",
+  "name": "IMAGE-browser",
```

Comment:
it shouldn't be changed here, instead please add a name field in package.json with name as "IMAGE-browser-extension". Package.json will automaticlly reflect that.

Reply:
@Cybernide from what I understand from the wireframe and our meetings, audio is the only one that is default right?